src/pages/user/authority/ConnectAuthority.tsx

```diff
@@ -1,20 +1,26 @@
 import React, { useState, useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { getAuthorities } from '../../../api/external/authorityRegisterApi';
+import { getAuthorities, updateAuthority } from '../../../api/external/authorityRegisterApi';
 import { RootStore } from '../../../redux/reducers/rootReducer';
 import styled from 'styled-components';
 import { useTranslation } from 'react-i18next';
 import { Authority } from '../../../types/authority.types';
 import AuthorityCard from './AuthorityCard';
+import { Button } from '@material-ui/core';
 
 const StyledAuthorityContainer = styled.div`
   > * {
     margin-top: 1rem;
   }
 `;
 
+const StyledClickableDiv = styled.div`
+  cursor: pointer;
+`;
+
 export const ConnectAuthority: React.FC = () => {
   const [matchingAuthorities, setMatchingAuthorities] = useState<Authority[]>([]);
+  const [selectedAuthorityScn, SetSelectedAuthorityScn] = useState<string>('');
```

Comment:
hva er Scn?

Reply:
trenger ikke useState<string> når initial value er tom streng. useState skjønner at det er string

Reply:
liten s i setSelectedAuthorityScn

Reply:
er med på `selectedSystemControlNumber`

Reply:
Trenger ikke å angi type for useState når det er tom stregn mener du?
I så fall kan man sette en verdi som ikke er string senere, så jeg tenker det er greit å være eksplisitt her

Reply:
jeg mener at `useState("")` er det samme som `useState<string>("")`
fordi typen string blir inferred